compose/src/main/java/com/bottlerocketstudios/compose/resources/Dimens.kt

```diff
@@ -45,8 +48,11 @@ class Dimensions(
 val smallDimensions = Dimensions(
     grid_0_25 = 1.5f.dp,
     grid_0_5 = 3.dp,
+    grid_0_75 = 5.dp,
     grid_1 = 6.dp,
+    grid_1_25 = 8.dp,
```

Comment:
should be 7.5

Reply:
@BottleRocket-Colin  grid_7_5 or (7.5).dp? I'm confused again lol.

Reply:
comment above was specific about using dp.  I just missed it here.

Reply:
all of the smaller dimension values should be exactly 75% of the original value.